planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -560,6 +586,38 @@ void ObstacleAvoidancePlanner::publishDebugData(const Header & header) const
 
   time_keeper_ptr_->toc(__func__, "  ");
 }
+
+void ObstacleAvoidancePlanner::transformPathToOdomFrame(Path::SharedPtr path_ptr) const {
+    geometry_msgs::msg::TransformStamped transform_stamped =
+            map_odom_tf_buffer_->lookupTransform("odom", "map", tf2::TimePointZero);
+
+    path_ptr->header.frame_id="odom";
+    path_ptr->header.stamp = this->now();
+
+    for (auto & point: path_ptr->points) {
+        autoware_auto_planning_msgs::msg::PathPoint odom_path_point;
```

Comment:
This seems unused...

Reply:
Yes, you are right. again, forget to remove it.